azure-pipelines.yml

```diff
@@ -201,15 +201,20 @@ jobs:
     # Set the remote cache GCP key path
   - bash: |
       set -x -e
-      # This command builds all software and runs all unit tests that run on
-      # the host.
+      # This command builds all software and runs all unit tests that run on the
+      # host, with a few exceptions. We exclude //quality because that's the
+      # purview of the `slow_lints` job. We also exclude the tests from
+      # //third_party/riscv-compliance because they're already covered by
+      # `execute_fpga_tests_cw310`.
       export GCP_BAZEL_CACHE_KEY=$(bazelCacheGcpKeyPath)
       ci/bazelisk.sh test \
       --build_tests_only=false \
       --test_output=errors \
       --define DISABLE_VERILATOR_BUILD=true \
       --test_tag_filters=-broken,-cw310,-verilator,-dv \
-      //sw/...
+      -- //... \
```

Comment:
This builds all the verilator targets including hw/build.verilator_real (for which `DISABLE_VERILATOR_BUILD` can't work.

Reply:
hw:verilator_real could also probably benefit from a manual tag

Reply:
Meh, it's easy. Done!